README.md

```diff
@@ -77,9 +77,11 @@ export default {
 }
 ```
 
-## Valid extras
+## Valid Extras
 - `path`: Change page URL
-- `alias`: Add single or multiple aliases to page
+- `alias`: Add single or multiple aliases to page, Module supports two type of alias
+- - **Simple Alias**: This aliases defines as simple string. If `routerNativeAlias` is true simple aliases will add as router alias, see vue-router [docs](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias)
```

Comment:
These aliases are defined as simple strings. If `routerNativeAlias` is true simple aliases will be added as router alias, see vue-router [docs](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias).